tests/trainers/tensorflow_examples.py

```diff
@@ -0,0 +1,53 @@
+"""Basic tests for the Python 3.6 and 3.9 base container images,based on tensorflow."""
+import tensorflow as tf
+
+
+def tensorflow_example_1():
```

Comment:
An explanation what this actually does would be really great.

Reply:
Comments added inline and in the README file